### PR TITLE
[9.0] [MIG] partner_multi_company

### PR DESCRIPTION
**Pull request by pedrobaeza**
Relevant commit is 84fe301d5c4d97596230f4ddb1d6d6869f079aa7

cc @Tecnativa

### CODE REVIEW

**Review by lasley — APPROVED** [review]
code, no test

**Review by sergio-teruel — APPROVED** [review]

File: partner_multi_company/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    def _amend_company_id(self, vals):
+        if 'company_ids' in vals:
+            if not vals['company_ids']:
+                vals['company_id'] = False
+            else:
+                for item in vals['company_ids']:
+                    if item[0] in (1, 4):
+                        vals['company_id'] = item[1]
+                    elif item[0] in (2, 3, 5):
```
Comment (carlosdauden) [inline partner_multi_company/models/res_partner.py:31]: Condition don't coverage on test

**Review by carlosdauden — APPROVED** [review]
Minor comments

File: partner_multi_company/models/res_partner.py (hunk 1 above)
Comment (pedrobaeza) [reply partner_multi_company/models/res_partner.py:31]: Supposedly this is covered by the line https://github.com/OCA/multi-company/pull/33/files#diff-529f207f5b8f18283bc6ced813ddf648R64, but it seems to be converted. Let me check, and if not, I masked it that line.

Comment (pedrobaeza) [reply partner_multi_company/models/res_partner.py:31]: OK, covered. It was only activated on create, not on write.

**Comment by pedrobaeza** [comment]
Another one with 100% coverage and 3 approvals. Merging!

**Comment by lasley** [comment]
Woot it was so much easier reviewing your work than doing this all myself like originally planned. Thanks again @pedrobaeza 😄

**Comment by pedrobaeza** [comment]
You owe me a drink then :stuck_out_tongue:

**Pull request merged**